### PR TITLE
Feature/team page redesign valuepage

### PR DESCRIPTION
**Pull request by cding-at-speer**

### CODE REVIEW

File: src/view/pages/Team/components/ValueDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+`;
+const ContentTitle = styled.div`
+    font-family: 'Poppins-Medium';
+    font-size: 15px;
+`;
+
+const ValueDetailContentContainer = styled.div`
+    width:283px;
+    @media ${MediaQuery.tablet} {
+        width:369px;
```
Comment (konner-at-speer) [inline src/view/pages/Team/components/ValueDetail.tsx:31]: Let's try and follow our coding convention everywhere by ensuring css properties and their values are spaced correctly. For example:` width: 369px`;

File: src/view/pages/Team/sections/TeamValue.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+const Image = styled.img<{ src: string }>`
+    max-width:100%;
+    @media ${MediaQuery.tablet} {
+        max-width:100%;
+        position:absolute;
+        right:0;
+    }
+`;
+
+const LuesImage = styled.img<{ src: string }>`
```
Comment (konner-at-speer) [inline src/view/pages/Team/sections/TeamValue.tsx:86]: From my understanding, this is supposed to be `ValuesImage`. We'll have to coordinate with the design team on this image and confirm if this is the correct image

File: src/view/pages/Team/sections/TeamValue.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    image        : string;
+}
+
+export const TeamValue: React.FunctionComponent<TeamValueData> = ({ title, subtitle }) => 
+{
+    return (
+        <Container>
+            <TeamValueImageWrapper>
+                <Image src={IMAGES.teamValueWords} alt="team value words" />
+                <LuesImage src={IMAGES.teamValueLues} alt="team value words" />
```
Comment (konner-at-speer) [inline src/view/pages/Team/sections/TeamValue.tsx:20]: Let's make both these `alt` tags more descriptive while being concise. Alt tags are not just a fallback for images but also provide feedback for screen readers and are important for SEO. 

Example: The `alt` tag for team value words could be "Integrity, innovate, and impact Speer team values"

File: src/view/pages/Team/sections/TeamValue.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    justify-content: center;
+    align-items: center;
+    position:relative;
+    width:365px;
+    @media ${MediaQuery.tablet} {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        position:relative;
```
Comment (konner-at-speer) [inline src/view/pages/Team/sections/TeamValue.tsx:61]: This redundant since these css properties are already defined above

**Review by konner-at-speer — CHANGES_REQUESTED** [review]
Looks good! As I mentioned below, we'll have to coordinate with the design team on the "values" image and confirm if it is the right one.

Also, do we need to use a mobile version of the values image (images/team/mobile/team_value_lues-mobile.png)? Could we not scale it down using `width` and `height` properties?

File: src/view/pages/Team/components/ValueDetail.tsx (hunk 1 above)
Comment (cding-at-speer) [reply src/view/pages/Team/components/ValueDetail.tsx]: Fixed. Is that possible for something like a prettier to do this for us automatically?

File: src/view/pages/Team/sections/TeamValue.tsx (hunk 2 above)
Comment (cding-at-speer) [reply src/view/pages/Team/sections/TeamValue.tsx]: slack

File: src/view/pages/Team/sections/TeamValue.tsx (hunk 3 above)
Comment (cding-at-speer) [reply src/view/pages/Team/sections/TeamValue.tsx]: changed to the following
`<Image src={IMAGES.teamValueWords} alt="Integrity, innovate, and impact" />
                <LuesImage src={IMAGES.teamValueLues} alt="Speer team values" />`

File: src/view/pages/Team/sections/TeamValue.tsx (hunk 4 above)
Comment (cding-at-speer) [reply src/view/pages/Team/sections/TeamValue.tsx]: cleaned up

File: src/view/pages/Team/components/ValueDetail.tsx (hunk 1 above)
Comment (konner-at-speer) [reply src/view/pages/Team/components/ValueDetail.tsx]: Potentially! Would have to look into it

**Pull request merged**